tasks/blacklight.rake

```diff
@@ -30,7 +30,10 @@ namespace :blacklight do
   end
 
   namespace :internal do
-    task seed: ['engine_cart:generate'] do
+    desc 'Index seed data in test ap'
+    task :seed do
+      Rake::Task['engine_cart:generate'].invoke unless File.exist?(EngineCart.destination)
```

Comment:
This allows repeated runs of the blacklight server task with the same generated app.

Reply:
Doesn't engine cart already check that the app is the same and do nothing?

Reply:
It should. I saw it regenerate once, but perhaps things were in a weird state. Removing.